src/encoded/static/components/browse/MicroscopySearchView.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+import React from 'react';
+import memoize from 'memoize-one';
+import _ from 'underscore';
+import url from 'url';
+
+import { getAbstractTypeForType } from '@hms-dbmi-bgm/shared-portal-components/es/components/util/schema-transforms';
+import { SearchView as CommonSearchView } from '@hms-dbmi-bgm/shared-portal-components/es/components/browse/SearchView';
+import { console, isSelectAction } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['console' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)